### PR TITLE
RAC-610: Add TabBar and Pill component

### PR DESCRIPTION
**Pull request by StevenVAIDIE**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: front-packages/akeneo-design-system/src/components/Pill/Pill.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# Pill
+
+## Usage
+
+TODO
+
+### TODO
+
+TODO
```
Comment (ValentinMumble) [inline front-packages/akeneo-design-system/src/components/Pill/Pill.stories.mdx:21]: The Pill component is used to attract attention on a specific part of the UI, such as on a list or table row, or on a Tab component.

See: <Link to="Tab Component with Pill" />

File: front-packages/akeneo-design-system/src/components/index.ts [hunk 2]
```diff
@@ -22,6 +22,8 @@ export * from './ProgressIndicator/ProgressIndicator';
 export * from './Search/Search';
 export * from './SectionTitle/SectionTitle';
 export * from './SwitcherButton/SwitcherButton';
+export * from './TabBar/TabBar';
 export * from './Table/Table';
 export * from './Toolbar/Toolbar';
 export * from './Tree/Tree';
+export * from './Pill/Pill';
```
Comment (ValentinMumble) [inline front-packages/akeneo-design-system/src/components/index.ts:29]: ORDER PLEASE!!!!!! 🇩🇪

File: front-packages/akeneo-design-system/src/components/Pill/Pill.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  width: 10px;
+  height: 10px;
+  background-color: ${props => getColorForLevel(props.level, 100)};
+  border-radius: 50%;
+`;
+
+type PillLevel = 'warning' | 'danger';
+
+type PillProps = {
+  level?: PillLevel;
```
Comment (ValentinMumble) [inline front-packages/akeneo-design-system/src/components/Pill/Pill.tsx:15]:
```suggestion
  /**
  * The level of the Pill.
  */
  level?: PillLevel;
```

File: front-packages/shared/src/microfrontend/MicroFrontendDependenciesProvider.tsx [hunk 4]
```diff
@@ -37,23 +37,32 @@ const MicroFrontendDependenciesProvider = ({
...
-            }
-
-            return token[1] + parameters[token[3]];
-          default:
-            throw new Error(`Unexpected token type: ${token[0]}`);
-        }
-      })
-      .reverse()
-      .join('');
+    let queryString = parameters
```
Comment (ValentinMumble) [inline front-packages/shared/src/microfrontend/MicroFrontendDependenciesProvider.tsx:40]:
```suggestion
    const queryString = parameters
```

**Review by ValentinMumble — APPROVED** [review]

**Review by juliensnz — APPROVED** [review]

**Pull request merged**